modules/conflict_resolver/README.md

```diff
@@ -0,0 +1,65 @@
+# Conflict Resolver
+
+## Purpose
+
+The conflict resolver is intended to allow the resolution of conflicts
+(different data entry for a given field between the "single data
+entry" and "double data entry" variations) for instruments which
+have double data entry enabled.
+
+## Intended Users
+
+The conflict resolver is usually used by data entry staff to resolve
+double data entry conflicts.
+
+Some studies may have protocols which involve supervisors or committees
+deciding how to properly resolve conflicts, and different studies may
+have different policies of if the same or different data entry staff
+is responsible for conflict resolution, but such policies are not
+enforced by LORIS.
+
+## Scope
+
+The module only provides data entry conflict resolution and triggers
+the recalculation of fields which are dependent on the data that's
+been changed (ie. score fields, age, etc). It does not do the initial
+data entry on the instruments themselves, only the resolution.
+
+It is up to any given study's internal policy to determine if
+resolution should be done by the same person or a different person
+than the person who did the initial data entry and not enforced by
+LORIS.
+
+## Permissions
+
+Accessing the module requires the `conflict_resolver` permission.
+
+A user should only see conflicts at the sites at which the user is
+a member.
+
+## Configurations
+
+The `DoubleDataEntryInstruments` setting determines whether or not
+an instrument gets flagged for conflicts. This is not directly used
+by the module, but the base instrument class uses this to determine
+whether or not to populate the conflicts table upon data entry
+completion.
+
+## Interactions with LORIS
+
+The conflict resolver module determines which fields are in conflict
+by looking into the `conflicts_unresolved` table, which instruments
+must populate upon the "Data Entry" flag being set to Complete.
+This is done by the base `NDB_BVL_Instrument` class and shouldn't
+need to be specifically set up for any instrument (other than as
+mentioned in the configurations setting.)
```

Comment:
Should it be "configuration settings" or "configurations setting"?

Reply:
there's just the one "DoubleDataEntryInstruments" setting mention in the section above, but I agree it doesn't read very well. I'll try and figure out a way to reword it and do it at the same time as the changes for your other comment (once we've agreed on some wording for the other part..)

Reply:
updated slightly.. is this better?